dashboard/test/ui/features/initial_page_views3.feature

```diff
@@ -24,6 +24,7 @@ Feature: Looking at a few things with Applitools Eyes - Part 3
     Given I am on "http://studio.code.org/"
     When I open my eyes to test "<test_name>"
     And I am on "<url>"
+    And I dismiss the language selector
```

Comment:
do you know why this step wasn't necessary before we upgraded the eyes gem?

Reply:
I suspect it's not related to the gem upgrade, but rather that I was running these against my localhost, which is where the language selector was showing.  I didn't figure out the root cause, but suspect that these just don't get run against localhost very often.